src/scenes/ApplicationRouter/portal/Review/index.tsx

```diff
@@ -88,12 +122,20 @@ const ReviewView = () => {
     [applicationSections]
   );
 
-  return (
-    <ApplicationPage crumbs={crumbs}>
-      {selectedApplication?.status === 'In Review' && <ApplicationStatusSubmitted />}
-      {selectedApplication?.status !== 'Submitted' && <ReviewCard sections={nonPrescreenSections} />}
-    </ApplicationPage>
-  );
+  const renderContent = (application: Application | undefined) => {
+    switch (application?.status) {
+      case 'In Review':
+        return <ApplicationStatusInReview />;
+      case 'Not Accepted':
+        return <ApplicationStatusNotAccepted application={application} />;
+      case 'Waitlist':
+        return <ApplicationStatusWaitlist application={application} />;
```

Comment:
We will need one more case for ACCEPTED. You can likely reuse the prescreen success page with some text modifications. Hope that helps!